app/src/main/res/layout/list_requests_item.xml

```diff
@@ -16,7 +16,7 @@
         android:layout_marginTop="16dp"
         android:maxLines="1"
         android:ellipsize="end"
-        tools:text="Mentor: Isabel Costa"
+        tools:text="@string/mentor_Isabel_Costa"
```

Comment:
Since the tools:text only affects the preview this isn't required.

Reply:
@epicadk  if tools:text is not required then those changes will be undone.